common4j/src/main/com/microsoft/identity/common/java/interfaces/IPlatformComponents.java

```diff
@@ -22,24 +22,24 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.java.interfaces;
 
-import com.microsoft.identity.common.java.cache.IMultiTypeNameValueStorage;
-import com.microsoft.identity.common.java.crypto.IDevicePopManager;
+import com.microsoft.identity.common.java.WarningType;
 import com.microsoft.identity.common.java.crypto.IKeyAccessor;
 import com.microsoft.identity.common.java.exception.ClientException;
-import com.microsoft.identity.common.java.WarningType;
 import com.microsoft.identity.common.java.providers.oauth2.IStateGenerator;
+import com.microsoft.identity.common.java.strategies.IAuthorizationStrategyFactory;
 import com.microsoft.identity.common.java.util.IBroadcaster;
 import com.microsoft.identity.common.java.util.IClockSkewManager;
 import com.microsoft.identity.common.java.util.IPlatformUtil;
-import com.microsoft.identity.common.java.strategies.IAuthorizationStrategyFactory;
 
-import edu.umd.cs.findbugs.annotations.Nullable;
 import lombok.NonNull;
 
 /**
- * Common components for each platforms.
+ * An interface for providing components for each platforms.
+ *
+ * TODO: make getPopManagerLoader() and getStorageLoader() part of this interface.
```

Comment:
Once getPopManagerLoader() and getStorageLoader() are part of this interface, 
Then I assume that we would remove the `IPlatformComponents extends IPopManagerLoader, IStorageLoader`

Reply:
Eventually, we could expose IPopManagerLoader and IStorageLoader directly via this interface (as opposed to extending it) - as that would come with a breaking change (Planned to do this in a separate PR).